deploy/staging/chainlog-ui.yaml

```diff
@@ -0,0 +1,43 @@
+replicaCount: 1
+service:
+  name: chainlog-ui
+  port: 80
+  type: ClusterIP
+  containerPort: 80
+  tls:
+    enabled: true
+    issuerName: cloudflare
+image:
+  repository: 068992353948.dkr.ecr.eu-central-1.amazonaws.com/chainlog-ui-staging
+  pullPolicy: IfNotPresent
+  tag: staging
+serviceAccount:
+  create: false
+ingress:
+  enabled: true
+  host: chainlog-staging-k8s.makerdao.com
```

Comment:
Perhaps we can create staging.makerdao.com zone

Reply:
@eskp I think we should keep same levels of DNS zones we have on production and on staging.
Otherwise we will need to validate staging and production SSL certificates differently

Reply:
Ok let's try but I think my suggestion is safer in the long run and might worth the effort